package.json

```diff
@@ -2,6 +2,7 @@
   "engines": {
     "node": ">=14.0.0"
   },
+  "type": "module",
```

Comment:
Why "module"?

Reply:
Due to ESM warning with Vite.js. Check deploy logs.

Reply:
Oh, just checked. I think the error was coming from the line on `Banner.vue`
```js
import ref from 'vue'
```

Try deleting the `"type": "module"` to see if it triggers the same error.

Reply:
It was an another error. Will try your suggestion.